code/datums/spell.dm

```diff
@@ -444,6 +444,11 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell))
 
 	return
 
+// Normally, AoE spells will generate an attack log for every turf they loop over, while searching for targets.
+// With this override, all /aoe_turf type spells will only generate 1 log, saying that the user has cast the spell.
+/obj/effect/proc_holder/spell/aoe_turf/perform(list/targets, recharge, mob/user, make_attack_logs)
+	add_attack_logs(user, user, "cast the spell [name]", ATKLOG_ALL)
```

Comment:
Can also remove the target here. So just null as second param. It now shows up as if it used the spell on himself.

Maybe some more description about it being an AOE spell in the `what` could also be nice